src/main/kotlin/com/exactpro/th2/check1/rule/AbstractCheckTask.kt

```diff
@@ -214,13 +216,13 @@ abstract class AbstractCheckTask(
         if (!taskState.compareAndSet(State.CREATED, State.BEGIN)) {
             throw IllegalStateException("Task $description already has been started")
         }
-        LOGGER.info("Check begin for session alias '{}' with sequence '{}' timeout '{}'", sessionKey, sequence, timeout)
+        LOGGER.info("Check begin for session alias '{}' with sequence '{}' timeout '{}'", sessionKey, sequence, taskTimeout.timeout)
```

Comment:
Please, add information about message timeout as well (or you can simply insert the whole **taskTimeout** into the string)